server/conf/routes

```diff
@@ -17,6 +17,7 @@ GET     /securePlayIndex            controllers.HomeController.securePlayIndex()
 # A controller for pages for an admin to create and maintain programs
 GET     /admin/programs                                         controllers.admin.AdminProgramController.index(request: Request)
 GET     /admin/programs/new                                     controllers.admin.AdminProgramController.newOne(request: Request)
+GET     /admin/programs/:programId/view                         controllers.admin.AdminProgramController.view(request: Request, programId: Long)
```

Comment:
**suggestion (blocking):** rename `view` to `show` here and in the controller, following the [routing convention](https://docs.civiform.us/contributor-guide/developer-guide/development-standards?q=standards#html-routing-convention)